### PR TITLE
CAR schema extension

### PR DESCRIPTION
**Pull request by omkar-g**

### CODE REVIEW

File: schema-extension.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+# CAR schema extension
+
+The CAR schema can be extended by creating `get_schema_extension` method in `app.py` file of the connector and returning `SchemaExtension` object.
+Example can found in [reference connector](./connectors/reference_connector/app.py)
+
+`SchemaExtension` object constructor needs following variables:
+- \<UUID\> is the ID of the extension. It is unique for every extension and it never changes.
+- \<Connector Name\> is a name that makes it easier to see where this extension comes from.
+- \<Version Number\> is an integer number encoded as a string; whenever an extension is changed this number needs to be incremented.
```
Comment (delliott90) [inline schema-extension.md]: Is the version number of the extension or the target CAR schema?

File: guide-build-connectors.md [hunk 2]
```diff
@@ -12,6 +12,9 @@ Delete the server folder. This is only used to provide dummy data to demo the co
 ### `app.py`
 
 Update the arguments required to connect to the target data source in the `__init__` method. The example in the reference connector uses `server`, `username` and `password`.
+
+`get_schema_extension` method is `app.py` has an example how to to extend [core CAR schema](https://github.com/IBM/cp4s-car-schema/blob/master/doc/generated/importSchema_v2.json). If the connector **doesn't need to extend the schema `get_schema_extension` method should be deleted**.
```
Comment (delliott90) [inline guide-build-connectors.md]: The `get_schema_extension` method in `app.py` has an example on how to extend...

File: schema-extension.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
+# CAR schema extension
+
+The CAR schema can be extended by creating `get_schema_extension` method in `app.py` file of the connector and returning `SchemaExtension` object.
```
Comment (delliott90) [inline schema-extension.md]: The CAR schema can be extended by creating **a** `get_schema_extension` method in  **the** `app.py` file of the connector and returning `SchemaExtension` object.

File: schema-extension.md (hunk 1 above)
Comment (omkar-g) [reply schema-extension.md]: Its version of extension, I'll update the description to make it clearer.

**Review by delliott90 — APPROVED** [review]

**Pull request merged**